src/sentry_plugins/gitlab/client.py

```diff
@@ -45,7 +45,7 @@ def get_issue(self, repo, issue_id):
                 ),
                 params={
                     # XXX(dcramer): this is an undocumented API
-                    'iid': issue_id,
+                    'iids[]': issue_id,
```

Comment:
Could we request `/projects/:repo/issues/:iid` here, instead of `/projects/:repo/issues?iids[]=:iid`?

I think that'll also allow us to drop `[0]` below, as well as the `try`/`except IndexError` stuff.